runtime/caml/intext.h

```diff
@@ -100,11 +100,17 @@
 #define ENTRIES_PER_TRAIL_BLOCK  1025
 #define SIZE_EXTERN_OUTPUT_BLOCK 8100
 
+struct caml_extern_state;
```

Comment:
This declaration is not needed, right? `caml_extern_state` is only ever used inside `extern.c`.
```suggestion
```

Reply:
Fixed in c29102e